receiver/awscontainerinsightreceiver/internal/k8swindows/k8swindows.go

```diff
@@ -26,7 +26,7 @@ type K8sWindows struct {
 	logger          *zap.Logger
 	nodeName        string `toml:"node_name"`
 	k8sDecorator    stores.K8sDecorator
-	summaryProvider *kubeletSummaryProvider
+	summaryProvider *kubelet
```

Comment:
change the field name to kubeletProvider?

Reply:
Addressed.
I changed it to kubeletSummaryProvider, since `kubeletProvider` represents interface to kubelet and this represents just one thing from kubelet i.e. container/pods summary.